modules/cnv-deleting-hco-subscription.adoc

```diff
@@ -0,0 +1,27 @@
+// Module included in the following assemblies:
+//
+// * cnv/cnv_install/uninstalling-container-native-virtualization.adoc
+
+[id="cnv-deleting-hco-subscription_{context}"]
+= Deleting the KubeVirt HyperConverged Cluster Operator catalog subscription
+
+To finish uninstalling {ProductName}, delete your KubeVirt HyperConverged
+Cluster Operator catalog subscription.
+
+.Prerequisites
+
+* An active *KubeVirt HyperConverged Cluster Operator* catalog subscription
+
+.Procedure
+
+. Ensure that you are in the `kubevirt-hyperconverged` project. If not, select
```

Comment:
How do you ensure this? I think it's safer to tell them to switch to the project both times.